### PR TITLE
gnome-shell: update to 3.30.0

### PR DESCRIPTION
**Pull request by Cogitri**

### CODE REVIEW

**Comment by maxice8** [comment]
missed this that fixes i686

```
diff --git a/srcpkgs/mutter/patches/remove-Werror.patch b/srcpkgs/mutter/patches/remove-Werror.patch
new file mode 100644
index 0000000000..98c4a252fa
--- /dev/null
+++ b/srcpkgs/mutter/patches/remove-Werror.patch
@@ -0,0 +1,11 @@
+--- configure
++++ configure
+@@ -18616,7 +18616,7 @@ if test "$enable_compile_warnings" != no ; then
+     if test "$enable_compile_warnings" = error; then
+       case " $CFLAGS " in
+       *[\ \   ]-Werror[\ \    ]*) ;;
+-      *) CFLAGS="$CFLAGS -Werror -Wno-error=deprecated-declarations" ;;
++      *) CFLAGS="$CFLAGS -Wno-error=deprecated-declarations" ;;
+       esac
+     fi
+   fi
```

File: srcpkgs/gnome-shell/template [hunk 1]
```diff
@@ -1,22 +1,29 @@
 # Template file for 'gnome-shell'
 pkgname=gnome-shell
-reverts="3.30.0_1"
-version=3.28.3
-revision=2
+version=3.30.0
+revision=1
```
Comment (Gottox) [inline srcpkgs/gnome-shell/template]: needs to be 2, as _1 was reverted.

File: srcpkgs/mutter/template [hunk 2]
```diff
@@ -1,43 +1,31 @@
 # Template file for 'mutter'
 pkgname=mutter
-reverts="3.30.0_1 3.30.0_2"
-version=3.28.3
-revision=3
+version=3.30.0
+revision=1
```
Comment (Gottox) [inline srcpkgs/mutter/template]: Needs to be 3 as _1 and _2 are reverted

File: srcpkgs/gnome-shell/template (hunk 1 above)
Comment (Cogitri) [reply srcpkgs/gnome-shell/template]: Has already been taken care of

**Pull request merged**